src/ArticleList.jsx

```diff
@@ -1,22 +1,28 @@
 import React, { useState, useEffect } from 'react'
-import axios from 'axios'
 import ArticleCard from './ArticleCard'
+import { getArticles } from './utils/api'
 
 function ArticleList() {
   const [articles, setArticles] = useState([])
+  const [loading, setLoading] = useState(true)
 
   useEffect(() => {
-    axios.get('https://nc-news-42oi.onrender.com/api/articles')
-        .then((response) => {
-        setArticles(response.data.articles)
+    getArticles()
+        .then((articles) => {
+        setArticles(articles);
+        })
+        .then (() => {
```

Comment:
This is good! You dont need this extra .then block, you can put the setLoading(false) in the same .then block above where you're setting the articles, as when the articles are retrieved you can change the loading state.